css/types/image.json

```
@@ -2054,7 +2054,7 @@
               "firefox": {
                 "version_added": "88",
                 "notes": [
-                  "The <code>type()</code> function is not supported as an argument to <code>image-set()</code>. See <a href='https://bugzil.la/1695404'>bug 1695404</a>.",
+                  "Prior to Firefox 89, the <code>type()</code> function was not supported as an argument to <code>image-set()</code>.",
```

A couple style suggestions (I _still_ need to properly document them somewhere):

```suggestion
                  "Before Firefox 89, the <code>type()</code> function is not supported as an argument to <code>image-set()</code>.",
```

Also, can we put this note after the next one, since the next one is now applicable to more versions?